### DISCUSSION TITLE
--overwrite add to documentation examples for Jamf configuration

### DISCUSSION THREAD

**Discussion by 000011111111** [grahampugh/erase-install#323] [Ideas]
Greetings, 

I noticed that  --overwrite options were not mentioned in the documentation wiki. I only found it in my logs for policy failure. 
### log report
 [erase-install] ERROR: Invalid installer is present. Run with --overwrite option to ensure that a valid installer is obtained.
###

After updating my Execute command in Jamf to include --overwrite the problem went away. 

Below is the example I have in my current configuration. 

`/Library/Management/erase-install/erase-install.sh --force-curl --reinstall --update --current-user  --overwrite --depnotify`

What are folks' thoughts about adding some information about this option - argument to the wiki? 

Thanks,  ~B

**Comment by grahampugh** [comment]
https://github.com/grahampugh/erase-install/wiki/2.-Downloading-(caching)-the-latest-macOS-installer-app#--overwrite ?

**Reply by 000011111111** [reply]
@grahampugh I apologize for my mistake in not RTFM this better. Thank you for building this software.

**Comment by grahampugh** [comment]
Don't forget also the --help argument.
